client/packages/programs/src/JsonForms/components/Prescription/useDraftLines.tsx

```diff
@@ -0,0 +1,29 @@
+import { useEffect, useState } from 'react';
+import { uniqBy } from '@openmsupply-client/common';
+import { DraftPrescriptionLine } from 'packages/invoices/src/types';
+import { useHistoricalStockLines } from 'packages/system/src';
+import { createDraftPrescriptionLineFromStockLine } from 'packages/invoices/src/Prescriptions/api/hooks/utils';
+
+export const useDraftLines = (itemId: string | null) => {
+  const [draftLines, setDraftLines] = useState<DraftPrescriptionLine[]>([]);
+
+  const { data: historicalStockLines } = useHistoricalStockLines({
+    itemId: itemId ?? '',
+  });
+
+  useEffect(() => {
+    if (!historicalStockLines) return;
+
+    const stockLines = uniqBy(historicalStockLines.nodes, 'id').filter(
+      stockLine => !stockLine.onHold
+    );
+
+    setDraftLines(
+      stockLines.map(stockLine =>
+        createDraftPrescriptionLineFromStockLine({ stockLine, invoiceId: '' })
+      )
+    );
+  }, [historicalStockLines]);
```

Comment:
I'm not 100% clear on what is being fetched here, and why, but I assume it's correct as it's following the same as the prior `useDraftPrescriptionLines` hook.

Reply:
We only show the available stock lines before creating the prescription right? Like we don't need to consider lines that already exist on the prescription here?

Reply:
Using historical stock lines is probably surplus to requirements, seeing as we don't support backdated prescriptions in the JSON form component... I suppose we could?

But I think better not to change without explicit decision, user should probably confirm something in the UI before we go backdating their prescriptions, seems out of scope for 2.8 fix!